glimr/utils.py

```diff
@@ -1,4 +1,76 @@
 from ray.tune.search import sample
+import os
+import pandas as pd
+
+
+def get_top_k_trials(
+    dir, metric="softmax_balanced", k=10, drop_dups=True, config_filter=None
```

Comment:
Suggest not setting a default value for `metric`.

Alternatively, set default to `None` and infer from .json file as first key.

Reply:
Incorporated the latter into the function (`metric=None`)